doc/user_guide.md

```diff
@@ -455,6 +455,102 @@ The main importer of OFT accepts markdown files with the extensions `.md` and `.
 
 Elektrobit's SpecObject format is read from SpecObject files with the `.xml` extension.
 
+## Console Tracing Report
+
+The Console Tracing Report is the standard report format of OFT. Its main purpose is to quickly debug broken tracing links. In this section you learn how to read this report.
+
+Below you see a typical example of a requirement from a design document.
+
+    ok - 0/2>0>0/1 - dsn~cli.tracing.default-format~1 (impl, utest)
+    |
+    | The CLI uses plain text as requirement tracing report format if none is given as a parameter.
+    |
+    |<-- ( ) impl~cli.tracing.default-format-2215031703~0
+    |--> ( ) req~cli.tracing.default-output-format~1
+    |<-- ( ) utest~cli.tracing.default-format-3750270139~0
+
+Let's go through its elements one by one.
+
+The first line is the summary.
+
+It starts with the status of the requirement &mdash; ok in this case.
+
+> **ok** - 0/2>0>0/1 - `dsn~cli.tracing.default-format~1` (impl, utest)
+
+Next we have a couple of numbers.
+
+The first pair shows how many of the broken incoming links this requirement has (zero), and how many in total (two).
+
+> ok - **0/2**>0>0/1 - `dsn~cli.tracing.default-format~1` (impl, utest)
+
+The number in the middle tells you how many duplicates this requirement has (also zero).
+
+> ok - 0/2>**0**>0/1 - `dsn~cli.tracing.default-format~1` (impl, utest)
+
+Consequently the next pair informs you how many (zero) of the overall (one) outgoing links are broken.
+
+> ok - 0/2>0>**0/1** - `dsn~cli.tracing.default-format~1` (impl, utest)
+
+The [Specification Item ID](#specification-item-id) in the middle is the unique technical ID of this requirement.
+
+> ok - 0/2>0>0/1 - **`dsn~cli.tracing.default-format~1`** (impl, utest)
+
+In the brackets you find, which artifact types this item expects as coverage. If the type is covered correctly, you see just the name there. 
+
+> ok - 0/2>0>0/1 - `dsn~cli.tracing.default-format~1` (**impl, utest**)
+
+If it is not covered, the name is lead in by a minus:
+
+> ok - 0/2>0>0/1 - `dsn~cli.tracing.default-format~1` (**-impl**, utest)
```

Comment:
I think the example should show not ok in this case

Reply:
Fixed.